validator/client/validator.go

```diff
@@ -120,6 +112,57 @@ func (v *validator) NextSlot() <-chan uint64 {
 	return v.ticker.C()
 }
 
+// LogValidatorGainsAndLosses logs important metrics related to this validator client's
+// responsibilities throughout the beacon chain's lifecycle. It logs absolute accrued rewards
+// and penalties over time, percentage gain/loss, and gives the end user a better idea
+// of how the validator performs with respect to the rest.
+func (v *validator) LogValidatorGainsAndLosses(ctx context.Context, slot uint64) error {
```

Comment:
Can this go into its own file? We have been doing good of keeping specific responsibilities in specific files and this feels like a monitoring type function.